plugins/dac_data_manager.c

```diff
@@ -1541,35 +1541,23 @@ static void manage_dds_mode (GtkComboBox *box, struct dds_tx *tx)
 	min_scale = manager->lowest_scale_point;
 	scale_available_mode = manager->scale_available_mode;
 
-	GtkWidget *tx1, *tx2 = NULL, *tx3 = NULL, *tx4 = NULL;
```

Comment:
I like this patch :)

I can't tell if this could break things, because it seems some code paths are not common to both dac1 & dac2 ; so, this patch also may hide some weird issues.
But the overall benefit of generalizing things helps and we can fix things later when we find them

Reply:
maybe make `dac1` & `dac2` into a `dac[]` array later ?
this doesn't need to be done now, but it's thought

if there is a patch that is already doing this, i'll get to it later and come back

Reply:
Yes, this could be the next step to make dac_data_manager more generic.